app/src/androidTest/java/com/github/se/cyrcle/ui/theme/ThemeTest.kt

```diff
@@ -21,22 +20,21 @@ class ThemeTest {
   @Composable
   fun ColoredSquare(
       color: Color,
-      colorScheme: ColorScheme = LightColorScheme,
       testTag: String = "TestTag",
   ) {
     Box(modifier = Modifier.size(100.dp).background(color).testTag(testTag)) {}
   }
 
   @Test
   fun disabledColorTestLight() {
-    composeTestRule.setContent { ColoredSquare(disabledColor(), LightColorScheme) }
+    composeTestRule.setContent { ColoredSquare(disabledColor()) }
     composeTestRule.onNodeWithTag("TestTag").assertIsDisplayed()
     // No test for checking the color
   }
 
   @Test
   fun disableColorTestDark() {
```

Comment:
Quick comment, there is another duplicated test here that you could delete!

Reply:
I was passing by, but could you check if you can remove the whole "part of color testing ? I never knew if it was really counted toward the code coverage ...